testing/testmedia.cpp

```diff
@@ -265,6 +266,13 @@ void SrtCommon::InitParameters(string host, string path, map<string,string> par)
                     cc.weight = stoi(check.queryValue("weight"));
                 }
 
+                if (check.parameters().count("source"))
```

Comment:
This fix is provided kinda "out of band", as I found this setting possible and lacking in the application support. The extras for socket options will be provided here in the same place.